packages/perseus/src/widgets/__stories__/interactive-graph.stories.tsx

```diff
@@ -135,6 +136,19 @@ export const AllLockedRays = (args: StoryArgs): React.ReactElement => (
     />
 );
 
+export const LockedCircle = (args: StoryArgs): React.ReactElement => (
+    <RendererWithDebugUI
+        apiOptions={{
+            flags: {
+                mafs: {
+                    segment: true,
+                },
```

Comment:
Let's use the `flagsForApiOptions` helper here.  That way as we flip more flags to true overall, we just update them in one place. 

https://github.com/Khan/perseus/blob/main/packages/perseus-editor/src/__stories__/flags-for-api-options.ts

Reply:
I'll make sure to do that in the editor, but I don't think that necessary for viewing locked features. Also, I think `flags-for-api-toptions.ts` is in the `perseus-editor` package, whereas this stories file is in the `perseus` package.